ocaml/xapi/xapi_observer.ml

```diff
@@ -736,3 +656,5 @@ let set_components ~__context ~self ~value =
       to_add
   in
   do_set_op ~__context ~self ~observation_fn ~db_fn
+
+let dummy_enabler () = true
```

Comment:
This name is not exposed in the commit

Reply:
Thanks, I missed removing this. Removed here https://github.com/xapi-project/xen-api/pull/5345/commits/cc2487298d1be6433d2eee553deaaffe9dac808b.